cli/template/extras/src/server/db/index-drizzle/with-postgres.ts

```diff
@@ -0,0 +1,12 @@
+import { drizzle } from "drizzle-orm/node-postgres";
+import { Client } from "pg";
```

Comment:
which one is more popular here? i've personally used postgres.js (postgres) more than node-postgres (pg)

Reply:
ps is more popular, but postgres.js is smaller and faster, I will switch to it here.